monai/data/dataset.py

```diff
@@ -1024,6 +1027,8 @@ def __init__(
         self._round: int = 1
         self._replace_done: bool = False
         self._replace_mgr: Optional[threading.Thread] = None
+        if runtime_cache:
+            raise NotImplementedError("runtime_cache=True not implemented.")
 
         super().__init__(data, transform, cache_num, cache_rate, num_init_workers, progress, copy_cache, as_contiguous)
```

Comment:
Could you please help also set `runtime_cache` value to the `super().__init__()`? In case we changed the default values in the `CacheDataset`.

Thanks.